lib/theme-i18n.json

```
@@ -45,6 +45,13 @@
 				}
 			]
 		},
+		"shadow": {
+			"presets": [
+				{
+					"name": "Shadow presets name"
```

```suggestion
					"name": "Shadow preset name"
```

Let's match the naming pattern

---

Can we perhaps use `Shadow name` instead? That's inline with what other presets do:

- font sizes: "font size name"
- colors: "color name"
- aspect ratios: "aspect ratio name"

---

Also: these changes need to be backported to wordpress-develop so that the wp-cli uses them ([see](https://github.com/wp-cli/i18n-command/blob/ebc79f4172fc69f84716885c3c50cbbc3e8cd293/src/JsonSchemaExtractor.php#L18)). Until then, these strings won't be picked up from anywhere.

---

@oandregal , I created a [diff](https://github.com/WordPress/wordpress-develop/compare/trunk...d-alleyne:wordpress-develop:fix/translatable-strings-in-theme-json) for wordpress-develop, but it instructs that the description should have a trac number. Do we have one, or do I create one?

---

You can use https://meta.trac.wordpress.org/ticket/7866 I've left a comment linking to this PR and the issue.

---

@oandregal I had created https://core.trac.wordpress.org/ticket/62728. I hope it isn't an issue.

---

That's good as well 👍